main.py

```diff
@@ -71,7 +71,7 @@ def get_random_app(self) -> dict:
 
 async def udb_cache_loop():
     while True:
-        url = "https://raw.githubusercontent.com/Universal-Team/db/master/docs/data/full.json"
+        url = "https://db.universal-team.net/data/full.json"
         resp = await app.state.session.get(url)
         r = json.loads(await resp.text())
```

Comment:
This should use ClientResponse.json() instead. i.e. `r = await resp.json()`

Reply:
Amended.